htetl/extract/phones.py

```diff
@@ -50,11 +51,29 @@ def _norm(text):
 
     return '-'.join([norm_area_code, norm_first_three, norm_last_four])
 
+
+def ExtractPhones(df):
+    '''
+        Does parsing logic
+    '''
+    phones =[]
+    for i, row in df.iterrows():
+        match = NUMBER_REGEX.findall(row["content"])
+
+        if match:
+            for phone in match:
+                pnumber=normalize(phone)
+
+                phones.append([row["id"],pnumber])
+
+    phonedf = pd.DataFrame(phones,columns=["pageid","phone"]).drop_duplicates()
+    return phonedf
+
 class ParsePhones(luigi.Task):
```

Comment:
Maybe move this into it's own module under `tasks`?

Reply:
The entire method or just the conversion to the dataframe? Extract the phone numbers in one task and normalize it in another?

Reply:
I'd move the class `ParsePhones` under `tasks`, similar to how I organized the external site extraction.